src/cpustat.hpp

```diff
@@ -15,9 +15,9 @@ void getHostname(std::string &hostname) {
 class CpuUsage {
 
 public:
-  double get_curr_val();
+  double get_curr_val() const;
```

Comment:
this will ensure that `get_curr_val()` does not change any `CpuUsage` instances!